Dockerfile.xdmod

```diff
@@ -0,0 +1,51 @@
+FROM moc-xdmod-base
+
+#COPY assets /tmp/assets
+#RUN /tmp/assets/copy-caches.sh
+#COPY bin /root/bin
+
+WORKDIR /root
+
+
+# This installs a barebones 10.0 instance of XDmod
+RUN mkdir -p /root/rpmbuild/RPMS/noarch \
+    && mkdir /tmp/supremm \
+    && yum -y install openssh-server openssh-clients pcp pcp-manager pcp-conf pcp-libs python-pcp perl-PCP-PMDA pcp-system-tools pcp-pmda-gpfs pcp-pmda-lustre pcp-pmda-infiniband pcp-pmda-mic pcp-pmda-nvidia-gpu \
+    pcp-pmda-nfsclient pcp-pmda-perfevent pcp-pmda-json python3 \
+    && pip3 install --upgrade pip \
+    && pip3 install keystoneauth python-keystoneclient python-novaclient python-cinderclient cryptography python-openstacksdk mysql-connector-python
+
+# RPM install of XDMod
+RUN cd /root/rpmbuild/RPMS/noarch \
+    && wget -nv https://github.com/ubccr/xdmod/releases/download/v10.0.0/xdmod-10.0.0-1.0.el7.noarch.rpm \
+    && yum -y install xdmod-10.0.0-1.0.el7.noarch.rpm
+
+COPY ./moc_xdmod_conf/httpd.conf /etc/httpd/conf/httpd.conf
+# am copying the xdmod-httpd conf file to ssl.conf as both use *:443
+# and this is easier than just deleting the ssl.conf
+RUN rm -f /etc/httpd/conf.d/ssl.conf \
+    && rm -f /run/httpd/httpd.pid
+
+COPY ./test_data/openstack.2018-04-17T00-00-00_2018-04-30T23-59-59.json /root/test/openstack/2018-04-17T00-00-00_2018-04-30T23-59-59.json
```

Comment:
I wonder if test data should just be mounted in a runtime rather than baked into the image?

Reply:
I'll move it to the "dev" build.

Reply:
Actually, I can eliminate this as I can copy in the test data with oc rsync.